part1/chapter6/quiz1/questions/question_2.rb

```diff
@@ -20,3 +20,35 @@
 # * Note: You can assume the players will only ever input `rock`,
 #   `paper` or `scissors`.
 
+paper = "paper"
+rock = "rock"
+scissors = "scissors"
+
+while true do
```

Comment:
A fine solution, but is the `while` loop necessary? You `break` out of the loop before it goes round again.